sklearn/metrics/tests/test_classification.py

```diff
@@ -324,6 +324,13 @@ def test_cohen_kappa():
     y2 = np.array([0] * 52 + [1] * 32 + [2] * 16)
     assert_almost_equal(cohen_kappa_score(y1, y2), .8013, decimal=4)
 
+    # Weighting example: none, linear, quadratic.
+    y1 = np.array([0] * 46 + [1] * 44 + [2] * 10)
+    y2 = np.array([0] * 50 + [1] * 40 + [2] * 10)
+    assert_almost_equal(cohen_kappa_score(y1, y2), .9315, decimal=4)
+    assert_almost_equal(cohen_kappa_score(y1, y2, weights="linear"), .9412, decimal=4)
+    assert_almost_equal(cohen_kappa_score(y1, y2, weights="quadratic"), .9541, decimal=4)
+
```

Comment:
The numbers were generate via package ml_metrics:

In[5]: import numpy as np
In[6]: from ml_metrics import kappa, linear_weighted_kappa, quadratic_weighted_kappa
In[7]: from sklearn.metrics import cohen_kappa_score
In[8]: y1 = np.array([0] * 46 + [1] * 44 + [2] * 10)
y2 = np.array([0] * 50 + [1] * 40 + [2] * 10)
In[9]: cohen_kappa_score(y1,y2)
Out[9]: 0.93150684931506866
In[10]: kappa(y1,y2)
Out[10]: 0.9315068493150684
In[11]: linear_weighted_kappa(y1,y2)
Out[11]: 0.9411764705882353
In[12]: quadratic_weighted_kappa(y1,y2)
Out[12]: 0.9541284403669725